dataclients/kubernetes/ingressv1.go

```diff
@@ -410,6 +410,12 @@ func (ing *ingress) ingressV1Route(
 	}
 	logger := newLogger("Ingress", i.Metadata.Namespace, i.Metadata.Name, loggingEnabled)
 
+	ingressValidator := &definitions.IngressV1Validator{}
+	if err := ingressValidator.Validate(i); err != nil {
+		logger.Errorf("Ingress validation failed: %v", err)
+		return nil, nil // Drop the resource without failing LoadAll
```

Comment:
If we returned and error here LoadAll will fail which doesn't make sense, so either move on with this or think of another solution.